Add Working with design files + heading update (#1187)
sourcegraph/about: handbook/product/design/design-process.md

## Process segments

> - **Ideation**

Reviewer: Should these sections be h3s instead? 
```suggestion
### Ideation
```

      - Schedule a meeting with engineers to discuss the interaction
  - Tools
      - Figma
- **Metrics collection**
  - In the RFC process, the definition of success will include items which can be measured and evaluated
  - Involve the business operations team in determining what metrics to track and how they are tracked
  - Work with engineers to add the required eventLog tracking
  - Tools
      - Looker
      - Google analytics
- **Refinement**
  - Designers will review metrics and determine if their designs are meeting objectives set for them.
  - If areas for improvement are found, tickets will be created to document the change required to improve the issue
  
## Working with design files

> To provide consistency in design files handling, follow the rules listed below:

Reviewer: ```suggestion
To provide consistency in design files handling, follow the rules listed below.
```

> #### File location

Reviewer: Headers should not jump multiple headers, so this should be an h3.

```suggestion
### File location
```

All files should be created under the Sourcegraph organization in Figma. It is important to not work on Sourcegraph projects in the private files as designs should be accessible to all Team Members.

#### Naming convention

> - All the files containing current project work must follow this naming structure:
> 
> RFC or GitHub issue number + RCF or GitHub issue title + stage (Draft, Open to review, Final review or Approved).
> 
> - Keep the stage updated as you progress with your work. 

Reviewer: The formatting is a little confusing here since the non-bulleted items is technically a sub-bullet. What do you think of this suggestion?

```suggestion

All the files containing current project work must follow this naming structure:

> RFC or GitHub issue number + RFC or GitHub issue title + stage (Draft, Open to review, Final review or Approved).

Keep the stage updated as you progress with your work. 
```

#### Project Tools
Use [Project Tools](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/Project-Tools?node-id=0%3A1) to provide more information about the project directly in the Figma file. It will help all the visitors connect files to the related RFC or GitHub issue and understand better all the details about the design.

> - Project Card - displays basic information about project ex. reviewers, RFC and GitHub issue links, deadlines etc. It should be created for every file.

Reviewer: ```suggestion
- Project Card - displays basic information about the project. For example, reviewers, RFC and GitHub issue links, and deadlines. It should be created for every file.
```

- Project Introduction - necessary when sharing your prototype with the Team. It should contain details about the target audience and project goals, to provide good context before starting the review.

- Final Page - placed at the end of the prototype. It shows additional notes on the solution, points out the most important pages in the prototype and describes what feedback the designer is looking for. It also instructs reviewers on how to leave comments.

> #### File structure

Author: That looks good @rrhyne. I think not all of those pages ill be required for every file. Maybe we can add a note 'Use only pages that are necessary for your file'. What do you think about it?

> #### File structure

Reviewer 2: Perfect, thanks!
